frontend/viewdir/competencyPage.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "template.njk" %}
+
+{% block content %}
+    <div class="container">
+    <h1 id="compTitle"style="color:#575756">Competencies</h1>
```

Comment:
Style shouldn't be used, should be in Css

Reply:
Shouldn't resolve a conversation unless you've addressed the issue, or given an explanation of why you won't address it.

What is your response to Stephen's point?